task_pool.go

```diff
@@ -0,0 +1,141 @@
+package getty
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+)
+
+import (
+	log "github.com/AlexStocks/log4go"
```

Comment:
In the dubbogo/getty, log4go is no longer in use.

Reply:
Yeah, I change it at once.